deepfence_worker/processors/bulk_processor.go

```diff
@@ -3,15 +3,49 @@ package processors
 import (
 	"context"
 	"fmt"
+	"os"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/cenkalti/backoff/v3"
 	"github.com/deepfence/ThreatMapper/deepfence_utils/log"
+	"github.com/deepfence/ThreatMapper/deepfence_utils/utils"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j/db"
 )
 
+var (
+	wait    chan struct{}
+	breaker sync.RWMutex
+)
+
+func init() {
+	wait = make(chan struct{})
+
+	neo4j_port := os.Getenv("DEEPFENCE_NEO4J_BOLT_PORT")
+	neo4j_host := os.Getenv("DEEPFENCE_NEO4J_HOST")
```

Comment:
can we get `neo4j_host` and `neo4j_port` from `directory` this will break multi tenancy in TS

Reply:
Good point, however the whole process here is based on one tenancy.
We will need to change it for multi-tenancy.